test/controllers/users_controller_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
+
+class UsersControllerTest < ActionDispatch::IntegrationTest
+  include Devise::Test::IntegrationHelpers
+
+  def setup
+    @user = users(:one)
+    @other_user = users(:two)
+  end
+
+  test 'should send friend request to another user' do
```

Comment:
Why did we have to write this test here? Let's stick to testing CRUD actions in controller tests.

We have one method in users_controller (index, which per my comments in it needs some work) and we aren't testing it here.